packages/api/src/sequelize/migrations/2025-06-17_01_create-patient-cohort.ts

```diff
@@ -0,0 +1,69 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import * as shared from "../migrations-shared";
+
+const patientTableName = "patient";
+const cohortTableName = "cohort";
+const patientCohortTableName = "patient_cohort";
+
+const cohortIdColumn = "cohort_id";
+const patientIdColumn = "patient_id";
+const cohortIdIndex = `${patientCohortTableName}_${cohortIdColumn}_idx`;
+
+export const up: Migration = async ({ context: queryInterface }) => {
+  await queryInterface.sequelize.transaction(async transaction => {
+    await shared.createTable(
+      queryInterface,
+      patientCohortTableName,
+      {
+        id: {
+          type: DataTypes.UUID,
+          primaryKey: true,
+          allowNull: false,
+        },
+        patientId: {
+          type: DataTypes.STRING,
+          allowNull: false,
+          field: "patient_id",
+          references: {
+            model: patientTableName,
+            key: "id",
+          },
+          onDelete: "CASCADE",
+        },
+        cohortId: {
+          type: DataTypes.UUID,
+          allowNull: false,
+          field: "cohort_id",
+          references: {
+            model: cohortTableName,
+            key: "id",
+          },
+        },
+      },
+      {
+        transaction,
+        addVersion: true,
+      }
+    );
+
+    await queryInterface.addConstraint(patientCohortTableName, {
+      fields: [patientIdColumn, cohortIdColumn],
+      type: "unique",
+      name: `${patientCohortTableName}_${patientIdColumn}_${cohortIdColumn}_unique`,
```

Comment:
Since we're defining those at the top of the file, maybe add this there as well?